src/store/modules/order/actions.ts

```diff
@@ -34,9 +35,15 @@ const actions: ActionTree<OrderState, RootState> = {
       unidentifiedProductItems.push(item);
       return ;
     }).filter((item: any) => item);
-    const original = JSON.parse(JSON.stringify(items))
 
-    commit(types.ORDER_LIST_UPDATED, { items, original, unidentifiedProductItems });
+    const original = JSON.parse(JSON.stringify(items));
+    
+    items = items.reduce((itemsByPoId: any, item: any) => {
+      itemsByPoId[item.orderId] ? itemsByPoId[item.orderId].push(item) : itemsByPoId[item.orderId] = [item] 
+      return itemsByPoId;
+    }, {});
+    
+    commit(types.ORDER_LIST_UPDATED, {items, original, unidentifiedProductItems});
```

Comment:
I think we should improve name, maybe
```suggestion
    commit(types.ORDER_PURCHASEORDERS_UPDATED, {items, original, unidentifiedProductItems});
```